backend/app/assets/stylesheets/spree/backend/shared/_base.scss

```diff
@@ -10,6 +10,8 @@ body {
   display: none;
 }
 
-.no-padding-bottom {
-  padding-bottom: 0;
-}
+// No-margin and no-padding helper classes
+@each $side in top bottom left right {
+  .no-margin-#{$side}  {margin-#{side}: 0}
```

Comment:
Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon